src/main/content/cli.html

```diff
@@ -32,7 +32,8 @@
 
                 <p>
                     Appsody by itself points to a growing community of stacks (Container images pre-built with runtimes
-                    and frameworks). Kabanero provides integrated collections for a subset of these stacks. 
+                    and frameworks). Kabanero provides integrated collections for a subset of these stacks, which extend the images to
+                    include container build and deployment artifacts, such as Tekton pipelines.
                     These are known as featured collections. When using Appsody with Kabanero, you point Appsody at your Collections.
```

Comment:
talked with Alan and we should rephrase this paragraph to the below:

>Appsody points to a growing community of Stacks (Container images prebuilt with runtimes and frameworks). Kabanero takes a subset of these stacks and augments them with build and deploy technology to create a Collection. Collections enable Kabanero to build an application in a controlled manner. This controlled flow ensures the application meets security requirements and deploys into a RHOCP (Red Hat OpenShift Container Platform) project. When using Appsody with Kabanero, you point Appsody at your Collection Hub.

Reply:
Updated and pushed.

Reply:
thanks Sue!